src/image-gallery.scss

```diff
@@ -152,6 +152,15 @@ $ig-background-black: rgba(0, 0, 0, .4) !default;
   display: inline-block;
   padding-right: 5px;
 
+  div {
```

Comment:
We don't want to style all of our divs to have this style. Use `image-gallery-thumbnail-label`